_sources/overview.rst

```diff
@@ -821,20 +1125,7 @@ Hidden Unit Tests with Graphical Status
          myTests().main()
 
 ShowEval Trace Mode
--------------------
-
-.. showeval:: showEval_0
-   :trace_mode: true
-
-   eggs = ['dogs', 'cats', 'moose']
-   ~~~~
-
-   ''.join({{eggs}}{{['dogs', 'cats', 'moose']}}).upper().join(eggs)
-   {{''.join(['dogs', 'cats', 'moose'])}}{{'dogscatsmoose'}}.upper().join(eggs)
```

Comment:
This is what I thought would break.  All of the `{{` are interpreted as things that the template engine should populate.  To show the source for a showeval you need to add an invisible space between the `{{`. See https://qwerty.dev/whitespace/